README.md

```diff
@@ -28,7 +28,8 @@ This documentation assumes you have a Kubernetes cluster already available. For
 [Minikube](https://github.com/kubernetes/minikube) is a tool that makes it easy to run Kubernetes locally. Minikube runs a
 single-node Kubernetes cluster inside a VM on your laptop for users looking to try out Kubernetes or develop with it day-to-day. 
 The below steps apply to a minikube cluster - the latest version as of writing this documentation is 0.23.0. You must also have 
-kubectl configured to access minikube.
+kubectl configured to access minikube. Please note that the Tensorflow Serving Kubernetes deployment does not currently work with
```

Comment:
I'd just make this a separate point for now. "If you're using minikube, the Virtualbox/VMware drivers are recommended. There are known issues with the KVM/KVM2 driver and TensorFlow serving tracked in <issue>".